rainforest/direct_connect.go

```diff
@@ -0,0 +1,36 @@
+package rainforest
+
+// DirectConnectParams are the parameters the Rainforest Direct Connect api expects
+type DirectConnectParams struct {
+	PublicKey string `json:"public_key"`
+	ID        string `json:"id"`
+}
+
+// DirectConnectConnection is the response from Rainforest's Direct Connect api telling us how to connect
+type DirectConnectConnection struct {
+	ID              string `json:"id"`
+	ServerPublicKey string `json:"server_public_key"`
+	ServerPort      int    `json:"server_port"`
+	ServerEndpoint  string `json:"server_endpoint"`
+}
+
+// ConfigureDirectConnect tells Rainforest to set up a direct connect tunnel for us
```

Comment:
```suggestion
// ConfigureDirectConnect tells Rainforest to set up a Direct Connect tunnel for us
```

Call the function `SetupDirectConnectTunnel` then?